ui/pubspec.lock

```diff
@@ -67,6 +67,13 @@ packages:
     description: flutter
     source: sdk
     version: "0.0.0"
+  intl:
```

Comment:
should be removed after you remove the package from package.yaml and run flutter pub get